src/crimson/os/seastore/logical_child_node.h

```diff
@@ -42,7 +42,9 @@ class LogicalChildNode : public LogicalCachedExtent,
 protected:
   void on_replace_prior() final {
     child_node_t::on_replace_prior();
```

Comment:
Let's rename this to `lba_child_node_t::on_replace_prior()` to avoid confusions.